plugins/src/main/java/org/moskito/control/plugins/sms/budgetsms/AlternateStatusChangeSmsNotifier.java

```diff
@@ -0,0 +1,106 @@
+package org.moskito.control.plugins.sms.budgetsms;
+
+import org.moskito.control.core.status.StatusChangeEvent;
+import org.moskito.control.plugins.notifications.AbstractStatusChangeNotifier;
+import org.moskito.control.plugins.sms.budgetsms.util.ParameterStringBuilder;
+import org.moskito.control.plugins.sms.twilio.SmsContentBuilder;
+import org.moskito.control.plugins.sms.twilio.StatusChangeSmsNotifier;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedReader;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.Map;
+
+public class AlternateStatusChangeSmsNotifier extends AbstractStatusChangeNotifier<BudgetSmsMessagingNotificationConfig> {
```

Comment:
Why is package called budgetsms but the notifier Alternate? Shouldn't it also be called BudgetSMS...